semcore/pills/src/style/pills.shadow.css

```diff
@@ -63,9 +63,12 @@ SPill[disabled] {
   }
 }
 
-SPills[keyboardFocused],
+SPills[keyboardFocused]:not([behavior=tabs]) {
+  box-shadow: var(--intergalactic-keyboard-focus, 0px 0px 0px 3px rgba(0, 143, 248, 0.5));
+}
 SPill[keyboardFocused] {
   box-shadow: var(--intergalactic-keyboard-focus, 0px 0px 0px 3px rgba(0, 143, 248, 0.5));
+  z-index: 1;
```

Comment:
how it will be displayable if we'll use Pills in some modal or another layout element with z-index > 0 ?

Reply:
It should work like a charm because z-indexes are stacking from parent components. I am also totally not worrying about cases when something may visually overlay the pill because it's getting z-index only in focused state.